perf/lmbench.py

```diff
@@ -116,6 +116,8 @@ def test(self):
 
         os.chdir(self.sourcedir)
         build.make(self.sourcedir, extra_args='rerun')
+        build.make(self.sourcedir, extra_args='rerun')
```

Comment:
why two times , `rerun`

Reply:
@PraveenPenguin 
Wanted to see the aggregate of two runs ... `make see` shows the aggregate of the above two runs.

Reply:
ah sure